docs/content/en/install/eks/_index.md

```diff
@@ -40,24 +40,138 @@ On this infrastructure the K8ssandra installation will consist of the following
 * 1x node Grafana deployment
 * 1x node Reaper deployment
 
-Feel free to update the parameters used during this guide to match your target deployment. The following should be considered a minimum for production workloads.
+Feel free to update the parameters used during this guide to match your target deployment. This should be considered a minimum for production workloads.
+
+## Infrastructure and Cassandra recommendations
+
+While the section above includes infrastructure settings for **minimum** production workloads, performance benchmarks reveal a wider range of recommendations that are important to consider. The performance benchmark report, available in this [detailed blog post](https://k8ssandra.io/blog/articles/k8ssandra-performance-benchmarks-on-cloud-managed-kubernetes/), compared the throughput and latency between:
+
+* The baseline performance of a Cassandra cluster running on AWS EC2 instances -- a common setup for enterprises operating Cassandra clusters
+* The performance of K8ssandra running on Amazon EKS, Google GCP GKE, and Microsoft Azure AKS. 
+
+It's important to note the following additional AWS infrastructure settings used in the benchmark:
+
+* 8 to 16 vCPUs 
+  * r5 instances: Intel Xeon Platinum 8000 series. Cassandra workloads are mostly CPU bound and the core speed made a difference in the throughput benchmarks.
+* 32 GB to 128 GB RAM (we used 64 GB RAM during the benchmark)
+* 2 to 4 TB of disk space
+  * In the benchmark, we used 1x 3.4 TB EBS gp2 volume
+* 10k Input/Output Operations per Second (IOPS)
```

Comment:
Did we specify IOPS or were these just observed values?

Reply:
Right, I think the 10k IOPS were just observed values in Alex's benchmark report, https://k8ssandra.io/blog/articles/k8ssandra-performance-benchmarks-on-cloud-managed-kubernetes/.

Reply:
Updated in more recent commit.